config/parameters/truth/base.txt

```diff
@@ -0,0 +1,23 @@
+# Basic configuration information shared across all truth model initial
+# conditions.
+#
+
+# Time model shared initial conditions.
+
+truth.dt.ns  170000000
+
+# Leader spacecraft attitude and orbit shared initial conditions.
+
+truth.leader.m  5.0
+truth.leader.J  0.03798 0.03957 0.00688
+
+truth.leader.wheels.J      135.0e-7
+truth.leader.wheels.w_max  677.0
+
+# Follower spacecraft attitude and orbit shared initial conditions.
+
+truth.follower.m  5.0
+truth.follower.J  0.03798 0.03957 0.00688
+
+truth.follower.wheels.J      135.0e-7
+truth.follower.wheels.w_max  677.0
```

Comment:
Set of configuration constants shared across all simulations.

Reply:
A lot of these constants look like they'd also be part of the existing C++ GNC code. Can we use those for configuration?

Reply:
The constants are in the existing GNC code. However, in order to have the "truth" constants separate from flight software's set of constants we should keep them in this file separately. It'll allow us to vet the constants going to make sure the system doesn't exhibit vastly different behavior given small perturbations in some of the constants.